umh-core/tools/loganalyzer/errors.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2025 UMH Systems GmbH
```

Comment:
All of this code seems to miss unit tests ?

Reply:
yes, as this code is supposed to be a working tool to help better troubleshoot. it is not supposed to be "stable" or even "code reviewed".

it is inspired by this HN article: https://www.geoffreylitt.com/2025/07/27/enough-ai-copilots-we-need-ai-huds

we can also remove it, but then we miss a good opportunity to have a tool to analyse the umh-core logs